README.md

```diff
@@ -1,24 +1,152 @@
-# README
+<div align="center">
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+  <h3><b>Rails-Blog-App</b></h3>
 
-Things you may want to cover:
+</div>
 
-* Ruby version
 
-* System dependencies
+# 📗 Table of Contents
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Rails-Blog-App\] ](#-rails-blog-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
 
-* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Database creation
+# 📖 [Rails-Blog-App] <a name="about-project"></a>
 
-* Database initialization
+**[Rails-Blog-App]** is a website built with RUBY on RAILS which empowers the readers and give them quality content
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+<summary>Language</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- Creates Blog-post.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```sh
+ sudo apt install postgresql postgresql-contrib
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+``` sh
+git clone git@github.com:bhushan354/Rails-Blog-App.git
+```
+
+### Install
+
+Install this project with:
+``` sh
+bundle install
+```
+
+
+### Usage
+
+
+### Run tests
+
+To run tests, run the following command:
+``` sh
+  cd Rails-Blog-App
+```
+
+``` sh
+  rails s
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Bhushan Deshmukh**
+
+- GitHub: [@bhushan354](https://github.com/bhushan354)
+- LinkedIn: [@bhushan354](https://www.linkedin.com/in/bhushan-deshmukh-5777851b1/)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add create blog functionality]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/sagieramos/Rails-Blog-App/issues).
+
```

Comment:
[REQUIRED] For a professional README, please make that there are no broken links. The link to the issues page is broken.

- Kindly add a valid issues page URL to fix this issue

[OPTIONAL] The command for running the project is documented under the `Run Tests` sub-heading. For a well organized README it would be best if the commands are located under the `Usage` sub-heading.

![image](https://github.com/bhushan354/Rails-Blog-App/assets/38088818/50303306-9616-4ea5-9c53-1b45ce6f6be5)

Additionally, you can also include commands like `rails db:setup` for creating the database and running migration files. This ensures well-detailed step-by-step instructions on how to setup and run the project locally.